package.json

```diff
@@ -31,8 +31,11 @@
         "reset": "pnpm -r --include-workspace-root --parallel exec pnpm dlx rimraf dist node_modules",
 
         "dev-sample": "pnpm --filter ./sample/* -r --parallel dev",
-        "build-sample": "pnpm --filter ./sample/* -r --parallel build",
-        "serve-sample": "pnpm --filter ./sample/* -r --parallel serve-build"
+        "build-sample": "pnpm --filter=./sample/* -r --parallel build",
+        "serve-sample": "pnpm --filter ./sample/* -r --parallel serve-build",
```

Comment:
synthax is different between the 2 lines for the filter

Reply:
Thanks, I removed a filter from the second one before creating the PR, that's why. Fixed.